csrc/executor.cpp

```diff
@@ -845,7 +869,11 @@ std::vector<at::Tensor> FusionExecutor::allocOutputs(
           kernel->outputs()[out_i]->isA<TensorView>(),
           "Cannot allocate outputs that are not tensors.");
       auto output = kernel->outputs()[out_i]->as<TensorView>();
-      if (alias_indices.count((int)out_i) != 0) {
+      if (auto need_fill =
+              getOutputTensorForFillWithInputTensor(kernel->outputs()[out_i])) {
+        outputs.push_back(
```

Comment:
How does this fill the output of the scatter op? It seems it actually aliases the input tensor, so it becomes an in-place op?

Reply:
In the `getTensorForFillAnotherTensor` function, I clone the `need_fill_tv` and get a new tensor, push it to the `outputs`.

Reply:
Oh, I see.